### DISCUSSION TITLE
class based on a child group state

### DISCUSSION THREAD

**Discussion by herzaso** [tailwindlabs/tailwindcss#12991] [Help]
I was trying to give a class to a parent based on a child group state but for some reason named-group didn't work.

So this worked (arbitrary group):
```
has-[.group:hover]:<some class>
```

but this didn't (named group):
```
has-[.group/mygroup:hover]:<some class>
```
I also tried some other variants like escaping the slash or passing it a hex but nothing worked.

Anyone has an idea?

**Comment by wongjn** [comment]
Backslash escape seems to work: https://play.tailwindcss.com/X9upUsM9Cv
